src/styles/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$calendarWidth: 720px;
+$daySize: 100px;
+$dayGap: 1px;
+$calendarPadd: 10px;
+$month-length: 31;
```

Comment:
'$month-length' variable name should follow the camel case naming convention. Consider renaming it to '$monthLength'.